typescript/db/src/utils/stripe.ts

```diff
@@ -1,6 +1,25 @@
+import { WorkspacePlan } from "@labelflow/graphql-types";
+import { DEFAULT_WORKSPACE_PLAN } from "@labelflow/common-resolvers/src/const";
+import { getUnixTime, addDays } from "date-fns";
 import { isNil } from "lodash/fp";
 import Stripe from "stripe";
 
+const DAYS_OF_TRIAL = 14;
+
+const planToStripePriceId: Record<WorkspacePlan, string> = {
+  Community: process.env.STRIPE_COMMUNITY_PLAN_PRICE_ID ?? "",
+  Starter: process.env.STRIPE_STARTER_PLAN_PRICE_ID ?? "",
+  Pro: process.env.STRIPE_PRO_PLAN_PRICE_ID ?? "",
+};
+
+const getPriceId = (
+  metadata: Stripe.Emptyable<Stripe.MetadataParam> | undefined
+): string => {
+  if (metadata && !isNil(metadata.plan)) {
+    return planToStripePriceId[metadata.plan as WorkspacePlan];
```

Comment:
Before casting, it's necessary to check whether `metadata.plan` matches one of the possible values of `WorkspacePlan`.